hazelcast/serialization/api.py

```diff
@@ -1175,33 +1175,30 @@ class CompactReader(abc.ABC):
     """
 
     @abc.abstractmethod
-    def read_boolean(self, field_name: str) -> bool:
-        """Reads a boolean.
+    def get_field_kind(self, field_name):
```

Comment:
Can you add return type?

Reply:
Importing `FieldKind` in this file causes a circular import. I'll probably add it once we remove the CompactReader base class.